schemas/gazette-notice-schema.json

```diff
@@ -0,0 +1,171 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Notice",
+  "description": "A gazette notice",
+  "type": "object",
+  "properties": {
+    "issue": {
+      "description": "The issue in which the notice is published",
+      "$ref": "#/definitions/issue"
+    },
+    "identifier": {
+      "description": "The identifier of the notice",
+      "type": "string"
+    },
+    "title": {
+      "description": "The title of the notice",
+      "type": "string"
+    },
+    "description": {
+      "description": "A description of the notice",
+      "type": "string"
+    },
+    "body": {
+      "description": "The content of the notice",
+      "type": "string"
+    },
+    "classification": {
+      "description": "The type of the notice",
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
+    "subjects": {
+      "description": "The entities that the notice is about",
+      "type": "array",
+      "items": {
+        "$ref": "includes/entity.json"
+      }
+    },
+    "url": {
+      "description": "The URL of the notice",
+      "type": "string",
+      "format": "uri"
+    },
+    "media_type": {
+      "description": "The media type of the URL resource",
+      "type": "string"
+    },
+    "creator": {
+      "description": "The author of the notice",
```

Comment:
It's possible creator might have been a word I used, but I'm not sure it's the correct one. Probably author is better, or even 'placed_by'. Either way, I think we could do with clarification in the description.